### PR TITLE
[Chore] Fix compilation issues with updated web3j

### PR DESCRIPTION
**Pull request by kapke**
## Description
Fix compilation issues with updated web3j, ~make Circle compile all targets.~

### CODE REVIEW

**Review by lemastero — APPROVED** [review]

**Review by mmrozek — COMMENTED** [review]
The changes from `.circleci/config` should be reflected in buildkite config

**Comment by mirkoAlic** [comment]
> The changes from `.circleci/config` should be reflected in buildkite config
if you are referring to compile-all step, that is already there: https://github.com/input-output-hk/mantis/pull/707/files#diff-4bc7af90a8380d7ed64283e70a4d7f2dR15

**Review by mirkoAlic — CHANGES_REQUESTED** [review]
With latest pull from develop, you should re-generate nix dependencies, given the web3j version was downgraded, and this PR is assuming a newer one.

File: .circleci/config.yml [hunk 1]
```diff
@@ -50,12 +56,6 @@ jobs:
           command: ./test-ets.sh
           no_output_timeout: 1h
 
-      - run:
-          name: additional compilation & dist
-          # this step builds parts of the codebase which are not tested in CI
-          # so as to prevent compilation regression
-          command: $SBT benchmark:compile snappy:compile dist
```
Comment (lemastero) [inline .circleci/config.yml:57]: Maybe we could keep `dist` step?

Comment (mirkoAlic) [reply .circleci/config.yml:57]: https://github.com/input-output-hk/mantis/pull/707/files#diff-4bc7af90a8380d7ed64283e70a4d7f2dR84 i think is already on buildkite side

Comment (mirkoAlic) [reply .circleci/config.yml:57]: i think it was already included in buildkite: https://github.com/input-output-hk/mantis/pull/707/files#diff-4bc7af90a8380d7ed64283e70a4d7f2dR84

Comment (mirkoAlic) [reply .circleci/config.yml:57]: Is already included for buildkite pipeline: https://github.com/input-output-hk/mantis/pull/707/files#diff-4bc7af90a8380d7ed64283e70a4d7f2dR84
or you mean having it as a separate step?

**Review by mirkoAlic — APPROVED** [review]
:100:

**Pull request merged**